src/languages/clle/clRunner.ts

```diff
@@ -0,0 +1,106 @@
+import { ExtensionContext, Position, Range, Selection, TextDocument, commands, window } from "vscode";
+import { instance } from "../../instantiate";
+import { GlobalConfiguration } from "../../api/Configuration";
+import { EndOfLine } from "vscode";
+
+export function initialise(context: ExtensionContext) {
+  context.subscriptions.push(
+    commands.registerCommand(`code-for-ibmi.cl.runSelected`, async () => {
+      const editor = window.activeTextEditor;
+
+      if (editor) {
+        const document = editor.document;
+
+        if (document && document.languageId === `cl`) {
+          const connection = instance.getConnection();
+
+          if (connection) {
+            const selectedCommand = getCommandString(editor.selection, document);
+            
+            if (selectedCommand.range) {
+              editor.selection = new Selection(
+                new Position(selectedCommand.range.start, 0), 
+                new Position(selectedCommand.range.end, document.lineAt(selectedCommand.range.end).range.end.character)
+              );
+            }
+            
+            const commandResult = await connection!.runCommand({
+              command: selectedCommand.content,
+              environment: `ile`
+            });
+
+            if (commandResult) {
+              if (commandResult.code === 0 || commandResult.code === null) {
+                window.showErrorMessage(
```

Comment:
```suggestion
                window.showInformationMessage(
```

because I get this:

![image](https://github.com/halcyon-tech/vscode-ibmi/assets/13275072/d6c9458c-4b10-43e2-b6e7-e6e6eefa7788)